server/routes/index.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import users from './user';
+
+const route = express.Router();
+
+route.use('/users', users);
```

Comment:
what is `users`?, I've seen this somewhere and I'm asking again

Reply:
Users is a route. All routes that contain `/user/` will be routed to the `routes/users/` file. Doing it this way will allow us abstract the different route files for users, comments, articles etc